.gitignore

```diff
@@ -5,4 +5,5 @@ extractor/vendor
 .idea
 #*
 .DS_Store
-*.iml
\ No newline at end of file
+*.iml
+*~
```

Comment:
Good question :)
This means that people who are using Emacs not going to see un-saved-but-edited files in git